source/_patterns/04-components/menus/menu--mobile/_menu--mobile.scss

```diff
@@ -14,6 +14,7 @@ $menu-mobile-submenu-hover-color: #000 !default;
   .menu__item {
```

Comment:
Colors defined at the top of this file should be changed to follow other components, using colors from the 'colors' section of the design tokens file.